.github/workflows/main.yml

```
@@ -486,6 +486,7 @@ jobs:
             tmp/$name/
           chmod +x tmp/$name/wasm*
           tar czvf gh-release/$name.tar.gz -C tmp $name
+          sha256sum gh-release/$name.tar.gz > gh-release/$name.tar.gz.sha256
```

Thank you for putting this all together, very nice!
So following what you found that is already used in the wild, let's go with `*.sha256sum`, what do you say?
```suggestion
          sha256sum gh-release/$name.tar.gz > gh-release/$name.tar.gz.sha256sum
```

---

No worries. Very well!